dashboard/test/ui/features/step_definitions/steps.rb

```diff
@@ -1448,6 +1456,13 @@ def create_user(name, url: '/users.json', code: 201, **user_opts)
   }
 end
 
+# press keys allows React to pick up on the changes
+And(/I press the keys of the section code into "([^"]*)"$/) do |selector|
```

Comment:
I think calling this "I enter the section code ____ into ____" might be a little bit more readable when it's used.  (Also, where is this used?)

Reply:
Used on line 240 of the UI test file. Since the section code is randomly generated in the Ruby file (`steps.rb`), I have no way of accessing that info from the UI test unless I create this additional step that parses the section URL for the section code and uses "press keys" to enter it into the React form.

Reply:
Ah, I see.  I might still use "enter the section code into" instead of "press the keys of the section code into" for better readability where it's used?  Up to you.